packages/syft/src/syft/server/server.py

```diff
@@ -1717,17 +1722,32 @@ def create_initial_settings(self, admin_email: str) -> ServerSettings:
             ).unwrap()
 
 
-def create_admin_new(
+def create_root_admin(
```

Comment:
we should rename this to reflect the logic

Reply:
it also feels like there is a third option: the root admin exists, with this email

Reply:
Then it would fail the first check, root verify key user already exists